lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -45,14 +45,16 @@ mapping:
 
       database_connection:
         type: str
-        default: sqlite:///./database/universe.sqlite?isolation_level=IMMEDIATE
```

Comment:
Is the default intentionally removed here and elsewhere?

Reply:
Yes. It is reassigning in `config/__init__.py` to the same value but without assuming "database" as the default for "data_dir". If we are no set the default here (which we should, I think), I'd do it after we have the functionality of #8689 (or smth similar) in place. Because there are other similar cases.

Reply:
Got it, I'll merge after the conflict is resolved. Thanks!

Reply:
Thank you!!!
Resolving (this one and the following ones).